ocaml/xapi/xapi_vif.ml

```diff
@@ -14,6 +14,7 @@
 
 open Stdext
 open Listext
+open Xapi_network_attach_helpers
```

Comment:
There is no need to open this. Just call `Xapi_network_attach_helpers.assert_can_see_named_networks` below.